### PR TITLE
chore: add api_shortname and library_type to repo metadata

### PR DESCRIPTION
**Pull request by parthea**
Update .repo-metadata.json as required by go/library-data-integrity

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 # [Cloud Logging for Bunyan: Node.js Client](https://github.com/googleapis/nodejs-logging-bunyan)
 
-[![release level](https://img.shields.io/badge/release%20level-general%20availability%20%28GA%29-brightgreen.svg?style=flat)](https://cloud.google.com/terms/launch-stages)
```
Comment (minherz) [inline README.md:7]: The content of the `README.md` is autogenerated. This change would be reverted by the owlbot.
What is a reason for this change? We use the same shields in all client libraries.

**Review by minherz — COMMENTED** [review]
please, revert the change in README.md. if the change in "shields" is integrated with README auto-generation, please apply the relevant changes to the README template instead.

File: README.md (hunk 1 above)
Comment (parthea) [reply README.md:7]: This change to the README is generated automatically by owlbot from .repo-metadata.json. This was not a manual change. The templates have been updated so this change should be reverted soon.

File: .repo-metadata.json [hunk 2]
```diff
@@ -4,10 +4,12 @@
...
   "issue_tracker": "https://issuetracker.google.com/savedsearches/559764",
-  "release_level": "ga",
+  "release_level": "stable",
   "language": "nodejs",
   "repo": "googleapis/nodejs-logging-bunyan",
   "distribution_name": "@google-cloud/logging-bunyan",
   "api_id": "logging.googleapis.com",
-  "codeowner_team": "@googleapis/api-logging"
+  "codeowner_team": "@googleapis/api-logging",
+  "api_shortname": "logging-bunyan",
```
Comment (bcoe) [inline .repo-metadata.json:13]:
```suggestion
```

**Review by bcoe — APPROVED** [review]

**Pull request merged**